packages/cache/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@actions/cache",
-  "version": "1.0.1",
+  "version": "1.1.0",
```

Comment:
This is a pretty small change so I think a patch bump to version `1.0.2` is sufficient

Could you also update `package-lock.json`? There is a version number there as well.

Reply:
Oh indeed.

Reply:
It was changed with the latest commit.